packages/@aws-cdk/aws-ecs-patterns/lib/base/scheduled-task-base.ts

```diff
@@ -137,6 +144,11 @@ export abstract class ScheduledTaskBase extends CoreConstruct {
    * The CloudWatch Events rule for the service.
    */
   public readonly eventRule: Rule;
+  /**
+   * The security group to use for the ECS Task.
+   */
+  private readonly _securityGroups?: ISecurityGroup[];
+
```

Comment:
nit: Add newline before, remove newline after.
```suggestion
  
  /**
   * The security group to use for the ECS Task.
   */
  private readonly _securityGroups?: ISecurityGroup[];
```